prospector/datamodel/advisory.py

```diff
@@ -113,7 +113,7 @@ def extract_camelcase_tokens(text) -> "list[str]":
     # return["blaHlafaHlsafs"]
 
 
-def extract_path_tokens(text: str) -> "list[str]":
+def extract_path_tokens(text: str, strict_extensions: bool = False) -> List[str]:
```

Comment:
please add an explanation of the meaning of strict_extensions in the doc-block of the function

Reply:
The original implementation used to check the extensions, but at the last meeting, we discuss creating a more relaxed version. I kept this functionality as a safety measure so we could use it later if needed. I did not want to remove the existing function.